tests/test_idaapi.py

```diff
@@ -584,10 +583,13 @@ def test_function_names(kernel32_idb, version, bitness, expected):
         else "__BaseDllInitialize@12"
     )
 
-    if version > 500:
-        with pytest.raises(KeyError):
-            # this is issue #7.
-            _ = api.idc.GetFunctionName(0x689018E5)
+    if version >= 720:
+        assert api.idc.GetFunctionName(0x689018E5) == "_InternalFindAtom@12"
+    # elif version > 500:
```

Comment:
Can you try to remove this comment, since we now have the means to autotest this?

Reply:
I meant, what is the name for 6.x databases?

Reply:
OK

Reply:
for 6.x idb, `api.idc.GetFunctionName(0x689018E5) == "sub_689017d4"`

looks doesn't make sense.

it should be `"sub_{:08X}".format(123)`

Reply:
now ignore this Error raise

![image](https://user-images.githubusercontent.com/24983369/90725852-5e956200-e2f3-11ea-895b-90842a984ee7.png)

So sometimes there are inconsistencies